### DISCUSSION TITLE
How i can download DCS_Liberation v6 (Preview builds)

### DISCUSSION THREAD

**Discussion by Maveric33** [Khopa/dcs_liberation#2394] [Q&A] [answered]
Hello,

Question that may seem silly but I've gone through the Git with a fine tooth comb and I can't figure out how to access version 6 for download.

Is the 5.2 and not 6 considered in Preview?

Thanks in advance

**Comment by Starfire13** [comment, marked as answer]
Question was posted on Discord and answered there.
